client/js/render.js

```diff
@@ -72,6 +73,10 @@ function appendMessage(container, chanId, chanType, msg) {
 }
 
 function buildChatMessage(msg) {
+	if (utils.lastMessageId < msg.id) {
+		utils.lastMessageId = msg.id;
+	}
```

Comment:
It's kinda horrible to see such side-effect inside the function itself. Can it go where the message gets appended to the container instead?

Reply:
I can move it to `appendMessage`.

Reply:
That would be better I think yeah.